app/views/exchange_rates/index.html.erb

```diff
@@ -26,7 +26,20 @@
       <% end %>
     </p>
 
-    <p class="govuk-body">Data is available in online (HTML), CSV and XML formats.</p>
+    <% if @period_list.spot? %>
```

Comment:
Couldn't we just localise these?

Reply:
t("exchange_rates.period_list.#{@period_list.rate_type}.description")

Reply:
Yes, I could.
But I guess it's better to wait to see if Chris provides a description for the monthly rate, too.
Also, the locales will not allow us to remove the `if` conditions, since there are custom messages for the "Data is available..." block too.